server/routes/api/landing_page.py

```diff
@@ -455,23 +455,28 @@ def data(dcid):
                 del spec_and_stat[category][topic]
             else:
                 spec_and_stat[category][topic] = filtered_charts
-        if not spec_and_stat[category]:
-            del spec_and_stat[category]
+        # Don't delete a category if it is in the valid categories list.
+        if (not spec_and_stat[category]) and ("validCategories"
```

Comment:
Can we make a meaningful named boolean variable here for this long condition? some formatting like:

if (a and
    b and
    c and
    d):

Can make it easier to view

Reply:
Done